packages/widget/src/context/wallet.ts

```diff
@@ -31,6 +35,13 @@ export enum WalletContextKeys {
   SUBSTRATE_WALLET = 'substrateWallet'
 }
 
+export type ParachainId = number;
```

Comment:
this is not needed as ParachainID is already provided by `@buildwithsygma`

Reply:
solved